lib/omnibus/software.rb

```diff
@@ -263,6 +263,8 @@ def dependency(val)
     #
     # @option val [String] :cookie (nil)
     #   a cookie to set
+    # @option val [String] :authorization (nil)
+    #   an authorization header to set
```

Comment:
Any chance to update this documentation with the example? I'd have to scrape the barrel of my web header knowledge to remember how to set this. Maybe something like this?
```suggestion
    #   an authorization header to set, e.g. "Basic " + Base64.encode64("#{username}:#{password}")
```